### ISSUE TITLE
Cannot find module 'timers' with webpack electron and karma

### ISSUE THREAD

**Issue by popod** [petkaantonov/bluebird#1614]
Since `bluebird 3.6.0` my unit tests does not work anymore.

The error is

```
Electron 6.0.11 (Node 12.4.0) ERROR
  Uncaught Error: Cannot find module 'timers'
  at webpack:///app/node_modules/bluebird/js/browser/bluebird.js:29:513 <- index.js:10450:289
```

Tested with `bluebird 3.5.5` and all works fine. `3.6.0` and upper thrown this error.

Any idea of the problem ? Thanks

**Comment by Slapbox** [comment]
Exact same issue in exact same environment.

**Comment by storageddd** [comment]
Have the same issue. Using Quasar (Electron mode).

**Comment by Slapbox** [comment]
For anyone using `yarn` (probably also works for `npm`) add this to your `package.json`:


```js
  "resolutions": {
    "bluebird": "3.5.5"
  },
```

**Comment by petkaantonov** [comment]
Fixed in 3.7.1
